packages/turf-transform-scale/index.js

```diff
@@ -33,38 +33,36 @@ var getCoords = invariant.getCoords;
 module.exports = function (geojson, factor, fromCenter, mutate) {
     // Input validation
     if (!geojson) throw new Error('geojson is required');
-
-    if (geojson.type === 'Point')
-        return (mutate === false || mutate === undefined) ? geojson : JSON.parse(JSON.stringify(geojson));
-
+    if (typeof factor !== 'number' && factor === 0) throw new Error('invalid factor');
     if (!factor) throw new Error('factor is required');
-    if (factor && typeof factor !== 'number' && factor === 0) throw new Error('invalid factor');
+
+    // Default params
+    var isMutate = mutate === false || mutate === undefined;
+    var isCenter = fromCenter === true || fromCenter === undefined;
+    var origin;
 
     // Shortcut no-scaling
-    if (factor === 1)
-        return (mutate === false || mutate === undefined) ? geojson : JSON.parse(JSON.stringify(geojson));
+    if (factor === 1) return geojson;
```

Comment:
@DenisCarriere what about:
`    if (factor === 1 || geojson.geometry.type === 'Point' || geojson.type === 'Point') return geojson;`
Scaling a `Point` is pointless 😄